bitsail-connectors/connector-kafka/src/main/java/com/bytedance/bitsail/connector/kafka/option/KafkaWriterOptions.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022-2023 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.kafka.option;
+
+import com.bytedance.bitsail.common.annotation.Essential;
+import com.bytedance.bitsail.common.option.ConfigOption;
+import com.bytedance.bitsail.common.option.WriterOptions;
+
+import static com.bytedance.bitsail.common.option.ConfigOptions.key;
+import static com.bytedance.bitsail.common.option.WriterOptions.WRITER_PREFIX;
+
+public interface KafkaWriterOptions extends WriterOptions.BaseWriterOptions {
+  ConfigOption<String> KAFKA_SERVERS =
+      key(WRITER_PREFIX + "kafka_servers")
+          .noDefaultValue(String.class);
```

Comment:
should we use `BOOTSTRAP_SERVERS` instead because in class `KafkaProduce` we use that?

Reply:
Changing config name is difficult once you have many running production jobs, so I'd like to avoid changing config name unless it's really necessary.